src/app/authorization/login/login.component.css

```diff
@@ -0,0 +1,13 @@
+*{
```

Comment:
Why global style is given it the login component? Please make sure that only component's styling reside in the component CSS file. and please make it scss instead of css.